packages/synapse-react-client/src/components/dataaccess/AccessRequirementDashboard.integration.test.tsx

```diff
@@ -60,18 +59,20 @@ describe('AccessRequirementDashboard tests', () => {
     expect(await screen.findAllByRole('combobox')).toHaveLength(1)
     expect(await screen.findAllByRole('textbox')).toHaveLength(2)
     await screen.findByTestId(AR_TABLE_TEST_ID)
-    expect(mockAccessRequirementTable).toHaveBeenCalledWith(
-      expect.objectContaining({
-        nameContains: '',
-        relatedProjectId: undefined,
-        reviewerId: undefined,
-        onCreateNewAccessRequirementClicked: mockOnCreateNewAR,
-      }),
-      expect.anything(),
+    await waitFor(() =>
+      expect(mockAccessRequirementTable).toHaveBeenLastCalledWith(
+        expect.objectContaining({
+          nameContains: '',
+          relatedProjectId: undefined,
+          reviewerId: undefined,
+          onCreateNewAccessRequirementClicked: mockOnCreateNewAR,
+        }),
+        expect.anything(),
+      ),
     )
```

Comment:
Wait for the last call to this mocked component to have the expected props. Part of the flakiness may be that we aren't giving React enough time to get through its render loop